tests/ruby/RuboCopBearTest.py

```diff
@@ -1,57 +1,91 @@
 import os
+from queue import Queue
+
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.settings.Section import Section
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
 
 from bears.ruby.RuboCopBear import RuboCopBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
 
-good_file = """def good_name
```

Comment:
Yes, rubocop requires that we have a valid filename extension (`.rb`). Temporary files were created in the previous case when we had to use `verify_local_bear(...)` but without a valid extension. See the commit message.